library/Sexp/src/Juvix/Sexp/Structure/generator.lisp

```diff
@@ -263,71 +263,73 @@ and a rhs that may contain a guard, so no = is assumed for the rhs"
                        (body from-body))
                   "")))))))
 
-(generate-haskell "ArgBody" '("sexp" "sexp") nil)
+(defun frontend-types ()
+  (generate-haskell "Type" (repeat 3 "sexp") "type" :list-star t)
```

Comment:
If I'm reading this file correctly, a bunch of spaces were added at the beginnings of lines, and some of the definitions were re-ordered.  Is any of that necessary, or is this some kind of reformatting accident?  Or is the file auto-generated and some structure changes made it reformat/reorder everything?

Reply:
I made functions, and split it between a few files!

Only 2 forms differ both the handlers which were incorrect